src/adv.py

```diff
@@ -49,3 +61,35 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+directions = ['n','s','e','w']
+valid_verbs = ['take', 'get', 'drop','i', 'inventory']
+
+while True:
+    print(player.current_room)
+    print("======================")
+    cmd = input(f"\n Please pick an option from below \n 1. Enter  {','.join(player.current_room.valid_directions())} to move\n 2. Type 'take' or 'get' and the item name to take an item\n 3. Enter 'drop' and the item name to return an item.\n 4. Type 'i' or 'inventory' to see the inventory \n 4. Press 'q' to quit\n")
+    print("======================")
+   
+    if cmd=='q':
+        print("GoodBye")
+        exit(0)
+    elif cmd in directions:
+            player.move(cmd)
+    elif cmd.split(" ")[0] in valid_verbs:
```

Comment:
I like this approach, it's very clean. And bails out early in the case of bad input. If you were making a game with a larger number of commands (or wanted to attach some additional data to those commands), you could try using a dictionary here instead, though that's certainly not necessary here